nexus.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+var fluid = require("infusion"),
+    kettle = fluid.registerNamespace("kettle"),
+    path = require("path");
+
+require("kettle");
```

Comment:
I introduced a new file index.js that does the fluid.module.register. This file is then used by the nexus.js driver and the tests.